README.md

```diff
@@ -1,20 +1,44 @@
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+A private beta service for:
 
-## Getting Started
+- [406Mhz beacon](https://www.gov.uk/maritime-safety-weather-and-navigation/register-406-mhz-beacons) owners to register their details with the Maritime & Coastguard Agency
+- Search and rescue [Mission Control Centres](<https://en.wikipedia.org/wiki/Mission_control_centre_(Cospas-Sarsat)>) to retrieve information about beacons during distress signal activations
 
-First, run the development server:
+This repository contains the frontend application of the service. It is built using [NextJS](https://nextjs.org/) and based on the [GOV.UK Design System](https://design-system.service.gov.uk/).
+
+## Getting started
+
+### Dependencies
+
+To develop locally, you'll need:
+
+- [Node.JS](https://nodejs.org/en/)
+- [Docker](https://www.docker.com/)
+
+### Running the application
+
+First, install dependencies:
+
+```bash
+npm install
+```
+
+Then, start the development server:
 
 ```bash
 npm run dev
-# or
-yarn dev
 ```
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+### Running in Docker
+
+To run the application in a production-like environment:
```

Comment:
This is not entirely true, as the target for the `docker-compose` file is the dev app.  I am starting to question the usefulness of having that stage in there entirely.  For now I would actually update the [docker-compose target] (https://github.com/madetech/mca-beacons-webapp/blob/main/docker-compose.yml#L7) to `nextjs-app` as this will build the production version of the application and allow you to standup a representative version that's deployed in AWS